controller/src/main/java/io/pravega/controller/server/AuthResourceRepresentation.java

```diff
@@ -120,6 +120,6 @@ public static String ofReaderGroupInScope(String scopeName, String readerGroupNa
      */
     public static String ofKeyValueTableInScope(String scopeName, String kvtName) {
         Exceptions.checkNotNullOrEmpty(kvtName, "KeyValueTableName");
-        return String.format("%s/%s", ofStreamsInScope(scopeName), kvtName);
+        return String.format("%s/%s_kvt", ofStreamsInScope(scopeName), kvtName);
```

Comment:
you shouldnt add the suffix here. This is the authorization resource against which the auth rules will be set outside of pravega. 
the resource name being scope/tablename is sufficient.

Reply:
If there is a stream and kvtable with the same name inside the same scope, earlier code would generate the same "resource name" for both causing confusion.... Hence I think it would be good to have this suffix so we know which resource this is actually for.

Reply:
@ravisharda thoughts?

Reply:
Are tables direct members of the scope? 

If yes, it might help to disambiguate what the resource type is, for authorization purposes. Say, both a stream and a table have the same name `abc`. So, the resource representation for both shall be `myscope/abc`. So, if the admin sets up an ACL for a user account such that it has access to `myscope/abc`, it'll authorize the user to both the stream and the table, which i sundesirable. 

I'd suggest putting the `_kvt` as a prefix tag, instead of a suffix. And also simplifying the tag name. Something like: `_kvtable_abc`. 

On a side note, this also reminds me that we need a more consistent way of structuring these resource names. But, that's for another day.

Reply:
Yes. Tables are direct members of scope. 
Just like a scope can have streams under it, the same scope can also have KeyValueTables under it and there could be a stream and kvtable with the same name under a given scope.

Reply:
Based on an offline discussion, we have decided to use the auth resource string `%s/kvtable/%s` for KeyValueTables